add implementation considerations for the batch issuance of the credentials (#643)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

It is up to the Credential Issuer whether to update both the signature and the claim values, or only the signature.

## Batch Issuing Credentials

> When the `batch_credential_issuance` parameter is not present in the Issuer metadata, it is at the discretion of the Credential Issuer to determine the number of the Credentials issued in the Credential Response, regardless of number of proofs contained in the `proofs` parameter in the Credential Request.

Reviewer: I'm not really sure what we're trying to clarify here, but I think the "When" adds extra confusion as the rest of the sentence is true when `batch_credential_issuance` is absent too I believe. So this sentence would at least be clearer I think:

```suggestion
It is at the discretion of the Credential Issuer to determine the number of the Credentials issued in the Credential Response, regardless of number of proofs contained in the `proofs` parameter in the Credential Request.
```

Reviewer: This sentence would be a further simplification to the language that I think still conveys the same meaning:

```suggestion
The Credential Issuer determines the number of the Credentials issued in the Credential Response, regardless of number of proofs/keys contained in the `proofs` parameter in the Credential Request.
```
